ruby/cross_service_examples/item-tracker/spec/api_spec.rb

```diff
@@ -0,0 +1,20 @@
+ENV["APP_ENV"] = "test"
+
+require "hello_world"  # <-- your sinatra app
+require "rspec"
+require "rack/test"
+require_relative "../src/app"
+
+RSpec.describe "The HelloWorld App" do
```

Comment:
Check out https://www.betterspecs.org/ for best practices on RSpec. What you have is fine but typically you would describe your class/methods.